tools/src/launcher/Main.hs

```diff
@@ -322,16 +311,25 @@ writeWindowsUpdaterRunner runnerPath = do
 
 spawnNode
     :: (FilePath, [Text], Maybe FilePath)
-    -> Shell (ProcessHandle, Async ExitCode, FilePath)
+    -> IO (ProcessHandle, Async ExitCode, FilePath)
 spawnNode (path, args, mbLogPath) = do
-    echo "Starting the node"
+    putStrLn "Starting the node"
+    -- We don't explicitly close the `logHandle` here,
+    -- but this will be done when we run the `CreateProcess` built
+    -- by proc later is `system'`:
+    -- http://hackage.haskell.org/package/process-1.6.1.0/docs/System-Process.html#v:createProcess
     (logPath, logHandle) <- case mbLogPath of
         Just lp -> do
-            mktree (FP.directory lp)
-            (lp,) <$> appendonly lp
+            createDirectoryIfMissing True (directory lp)
+            (lp,) <$> openFile lp AppendMode
         Nothing -> do
             tempdir <- liftIO (fromString <$> getTemporaryDirectory)
-            mktemp tempdir "cardano-node-output.log"
+            -- FIXME (adinapoli): `Shell` from `turtle` was giving us no-resource-leak guarantees
```

Comment:
AFAIK Windows 7 doesn't clean up the temp dir, so it's better to delete the file, I think (otherwise logs will just accumulate and accumulate).